src/identity/identity.ts

```diff
@@ -145,6 +150,12 @@ export class Identity {
 		return this.hasRole(Role.LEARNING_ARCHIVE) || this.isSuperUser()
 	}
 
+	isReporter() {
+		return this.isCshrReporter() || this.isProfessionReporter() ||
+			this.isOrganisationReporter() || this.isKPMGSupplierReporter() ||
+			this.isKornferrySupplierReporter();
+	}
+
```

Comment:
Would be more efficient to use `hasAnyRole()` here?

```ts
this.hasAnyRole(Role.CSHR_REPORTER, Role.PROFESSION_REPORTER...)
```

Reply:
I don't think it'd be faster performance-wise. I prefer using functions as opposed to hardcoded roles, because if the names/enums of those roles change down the line it'd cause a greater change overall (in all of the places that you've used those hardcoded names). Interfacing it with a function keeps it simple to read whilst allowing the underlying role name/enum to be anything we want.